src/App.js

```diff
@@ -48,15 +52,15 @@ class App extends Component {
          />
          <Dropdown  
          label='Type'
-        options={typeOptionList}
-        thingHappen={this.handleType}/>
-         <PokeList thing= {this.state.data}/>
+         options={typeOptionList}
+         thingHappen={this.handleType}/>
+         <PokeList thing={filterType}/>
```

Comment:
While I understand using "thing" and "thing happen", you really want to try and name your variables and functions appropriately in a manner that accurately reflects their purpose.